### ISSUE TITLE
Peer Dependencies not satisfied

### ISSUE THREAD

**Issue by Secretmapper** [sstur/react-rte#41]
Seems like the upgrade to 0.6.0 draft-js broke the package, since the export/import-ers uses 0.5.0 as the draft-js peer dependency.

**Comment by sstur** [comment]
Oop, will fix.

**Comment by Secretmapper** [comment]
Any timeline on this @sstur ? I know you're probably busy, but it'd be really solid if you can get it fixed soon. I'd really rather not fork 4-5 packages to do a bunch of few line updates 😄

**Comment by sstur** [comment]
I'll try to get this sorted out in the next 12 hours. Is it just broken in master or on npm also?

**Comment by sstur** [comment]
I just rolled back the dependency in this repo from draft-js@0.6.0 to 0.5.0. Will that help? This has not been published to npm yet. Any idea what's the a better solution here? I guess what I want to say is that they all depend on 0.5.0 or greater.

**Comment by sstur** [comment]
Here's what I see on a fresh install of the current npm latest react-rte@0.1.14:

```
sstur@local:example4$ npm install react-rte
/Users/Me/projects/example4
├── UNMET PEER DEPENDENCY react@^15.0.1
├── UNMET PEER DEPENDENCY react-dom@^15.0.1
└─┬ react-rte@0.1.14 
  ├── classnames@2.2.5 
  ├─┬ UNMET PEER DEPENDENCY draft-js@0.4.0
  │ └─┬ fbjs@0.8.1 
  │   ├─┬ babel-plugin-syntax-flow@6.8.0 
  │   │ └─┬ babel-runtime@6.6.1 
  │   │   └── core-js@2.3.0 
  │   ├── core-js@1.2.6 
  │   ├─┬ isomorphic-fetch@2.2.1 
  │   │ ├─┬ node-fetch@1.5.1 
  │   │ │ ├─┬ encoding@0.1.12 
  │   │ │ │ └── iconv-lite@0.4.13 
  │   │ │ └── is-stream@1.1.0 
  │   │ └── whatwg-fetch@1.0.0 
  │   ├─┬ loose-envify@1.1.0 
  │   │ └── js-tokens@1.0.3 
  │   ├─┬ promise@7.1.1 
  │   │ └── asap@2.0.3 
  │   └── ua-parser-js@0.7.10 
  ├─┬ draft-js-export-html@0.1.13 
  │ ├── UNMET PEER DEPENDENCY draft-js@^0.5.0
  │ └── draft-js-utils@0.1.4 
  ├─┬ draft-js-export-markdown@0.1.5 
  │ └── UNMET PEER DEPENDENCY draft-js@^0.5.0
  ├─┬ draft-js-import-html@0.1.8 
  │ ├── UNMET PEER DEPENDENCY draft-js@^0.5.0
  │ └─┬ draft-js-import-element@0.1.11 
  │   └── UNMET PEER DEPENDENCY draft-js@^0.5.0
  ├─┬ draft-js-import-markdown@0.1.4 
  │ ├── UNMET PEER DEPENDENCY draft-js@^0.5.0
  │ └── synthetic-dom@0.1.4 
  ├── draft-js-tools@0.1.2 
  ├── immutable@3.8.1 
  ├── UNMET PEER DEPENDENCY react@^0.14.0 || ^15.0.0-rc
  └── UNMET PEER DEPENDENCY react-dom@^0.14.0 || ^15.0.0-rc

npm WARN react-rte@0.1.14 requires a peer of react@^15.0.1 but none was installed.
npm WARN react-rte@0.1.14 requires a peer of react-dom@^15.0.1 but none was installed.
npm WARN draft-js-export-html@0.1.13 requires a peer of draft-js@^0.5.0 but none was installed.
npm WARN draft-js-export-markdown@0.1.5 requires a peer of draft-js@^0.5.0 but none was installed.
npm WARN draft-js-import-markdown@0.1.4 requires a peer of draft-js@^0.5.0 but none was installed.
npm WARN draft-js-import-html@0.1.8 requires a peer of draft-js@^0.5.0 but none was installed.
npm WARN draft-js@0.4.0 requires a peer of react@^0.14.0 || ^15.0.0-rc but none was installed.
npm WARN draft-js@0.4.0 requires a peer of react-dom@^0.14.0 || ^15.0.0-rc but none was installed.
npm WARN draft-js-utils@0.1.4 requires a peer of draft-js@^0.5.0 but none was installed.
npm WARN draft-js-import-element@0.1.11 requires a peer of draft-js@^0.5.0 but none was installed.
```

Is this what you've been experiencing?

**Comment by sstur** [comment]
OK. I've pushed a new version. I believe that should fix your issues. Turns out peer dependencies are a strange thing when using npm for front-end projects. Purely by one of the dependencies of this project (draft-js-export-html for instance) bumping the versions in their package.json that can break this package. I feel that on server-side projects we would just use regular dependencies instead of peer dependencies but with bundlers like webpack that would mean multiple copies of the same thing gets included in the bundler. The standard way around this seems to be peer dependencies but it's suboptimal at best.
